src/styles/main.scss

```diff
@@ -1,3 +1,70 @@
+
+$cellSize: 100px;
+$gap: 1px;
+$calendarSize: calc(#{$cellSize} * 7 + #{$gap} * 6);
```

Comment:
I guess you can avoid using calc with scss, am I wrong?

btw, what is 7 and 6?

Reply:
I tried without calc but it didn't work, but I'll try once more.

7 is the amount of days in a row and 6 is the amount of gaps between them